server/controllers/article.controller.js

```diff
@@ -204,5 +210,37 @@ class ArticleController {
     await sendMail(mailOption);
     return Response(res, 200, 'Article shared successfully');
   }
+
+  /**
+   * @description Get top rated articles controller
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} Response
+   */
+  static async getTopArticle(req, res) {
+    const allarticles = await Article.findAll({
+      where: {
+        published: true,
+      },
+      attributes: {
+        exclude: ['published', 'userId']
+      },
+      include: [{
+        model: User,
+        as: 'author',
+        attributes: {
+          exclude: ['istokenreset', 'password', 'social', 'roleId', 'subscribed', 'id']
+        }
+      }]
+    });
+    const rate = allarticles.map(async (article) => {
+      const ratings = await getArticleRating(article.dataValues.id);
+      const rateObj = { article, ratings };
```

Comment:
The ratings value should a property in the same object of the article

Reply:
@geekdanitek Thanks would fix that